src/main/java/hudson/plugins/promoted_builds/Status.java

```diff
@@ -271,6 +271,12 @@ public Promotion getLast() {
         return null;
     }
 
+    public Boolean isLastAnError() {
```

Comment:
Would it be useful to make new public methods that only exist to serve the UI `@Restricted(NoExternalUse.class)`?